docs/modules/plugins/pages/plugin-azure-storage-blob.adoc

```diff
@@ -0,0 +1,112 @@
+= Azure Storage Blob Plugin
+
+The plugin provides functionality to interact with https://azure.microsoft.com/en-us/services/storage/blobs/[Azure Blob Storage]
+
+== Installation
+
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-azure-storage-blob', version: '{current-version}')
+----
+
+== Configuration
+
+=== Storage account configuration
+
+Storage accounts are configured with a set of properties with the following format:
+
+[source,properties]
+----
+azure.storage.blob.account.{account-name}.{property-name}=property value
+----
+
+Where:
+
+. `account-name` - The name of the storage account
+. `property-name` - The name of the account property. One of: 
+.. `name` - The name of the storage account
+.. `sas-token` - The https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview[SAS token]
```

Comment:
Why not to use Azure Active Directory as for other services? SAS token creation could be challenging due to permission restrictions